README.md

```diff
@@ -1,20 +1,20 @@
 # express-interceptor
 
->A tiny Express response interceptor.
+_A tiny Express response interceptor_
 
-Express-interceptor allows you to define a previous step before sending a response. This allows you to do anything you want with the response, such as processing, transforming, replacing, or logging it. Express-interceptor allows you to avoid the necessity of calling `next()` over and over. Further, you can avoid managing confusing nested scopes. Using a declarative API, it's simple to use and maintain. 
+Express-interceptor allows you to define a previous step before sending a response. This allows you to do anything you want with the response, such as processing, transforming, replacing, or logging it. Express-interceptor allows you to avoid calling `next()` over and over. Further more, you can avoid managing nested scopes. Using a declarative API, it’s simple to use and maintain.
 
 [![NPM](https://nodei.co/npm/express-interceptor.png)](https://nodei.co/npm/express-interceptor/)
 
 [![Build Status](https://travis-ci.org/axiomzen/express-interceptor.svg)](https://travis-ci.org/axiomzen/express-interceptor) [![Dependencies](https://david-dm.org/axiomzen/express-interceptor.png)](https://david-dm.org/axiomzen/express-interceptor.png)
 
-## Raison d'etre
+## Raison d’être
 
 We tested other packages that offer a similar solution, but found they are difficult to organize, to maintain logic, and to process responses before they are sent. We developed a solution to this problem that offers a declarative-structured way to organize code, and that defines custom middleware for response processing.
```

Comment:
I recommend writing the README in a more objective voice, “packages that offer similar solutions lack of ...”, instead of “we tested other packages...”. Also, in _raison d’être_, is more important to state the problem that the package solve, then the how (architectural differences, patterns) and _maybe_ mention that other packages lack of that (not necessary in my opinion).